README.md

```diff
@@ -24,13 +19,19 @@ This is found at `./static-site/`.
 See [static-site/README.md](./static-site/README.md) for instructions on how to add documentation and develop.
 This is built using Gatsby during deployment and served via `./server.js`.
 
-
+---
 ## Auspice
-Auspice is the software used to visualise & interact with phylogenomic data.
-The auspice customisations are found in `./auspice/client/`, with the server code for handling auspice-derived requests for datasets etcetera in `./auspice/server`.
+[Auspice](https://github.com/nextstrain/auspice) is the web-facing (e.g. "client") software used to visualise & interact with phylogenomic data.
+When nextstrain.org is built, it uses a JavaScript bundle from a previously installed version of auspice.
+By default, `nextstrain.org` is built using a globally installed version of auspice specified in [set-up.sh](/set-up.sh).
+To develop `nextstrain.org` using a local development version of auspice, do not run `set-up.sh`.
+Instead, follow [these auspice installation instructions](https://github.com/nextstrain/auspice/blob/master/docs-src/docs/installation.md#installing-from-source) to set up your desired auspice environment.
+
+### Auspice customisations
```

Comment:
Due to the folder structure placing both auspice client customisation code & the code related to handling auspice-derived GET requests, can we rename this "Auspice client customisations"